### PR TITLE
Updating to reflect meteor-integration pull request

### PR DESCRIPTION
**Pull request by jangerhofer**
Changing createApolloServer interface to reflect changes made [this pull request.](https://github.com/apollostack/meteor-integration/pull/26)

### CODE REVIEW

File: source/apollo-client/meteor.md [hunk 1]
```diff
@@ -34,19 +34,23 @@ Define your schema and resolvers, and then set up the Apollo server with [`creat
...
+  typeDefs,
+  resolvers
+});
+
+
 createApolloServer({
-  graphiql: true,
-  pretty: true,
   schema,
-  resolvers,
```
Comment (lorensr) [inline source/apollo-client/meteor.md]: what about resolvers?

**Comment by lorensr** [comment]
Looks good, thanks 😊

**Comment by jangerhofer** [comment]
With the https://github.com/apollostack/meteor-integration/pull/26 merge, is this update ready to roll?

**Comment by lorensr** [comment]
Once a couple people test and it's out of beta (currently `meteor add apollo` does `0.0.3`)

**Comment by jangerhofer** [comment]
Whoops -- my mistake! :)

**Comment by helfer** [comment]
Hey @lorensr feel free to merge this when it's ready!

File: source/apollo-client/meteor.md [hunk 2]
```diff
@@ -34,19 +34,23 @@ Define your schema and resolvers, and then set up the Apollo server with [`creat
...
 import { createApolloServer } from 'meteor/apollo';
+import { makeExecutableSchema, addMockFunctionsToSchema } from 'graphql-tools';
 
-import schema from '/imports/api/schema';
+import typeDefs from '/imports/api/schema';
 import resolvers from '/imports/api/resolvers';
 
+const schema = makeExecutableSchema({
+  typeDefs,
+  resolvers
```
Comment (rdickert) [inline source/apollo-client/meteor.md]: style: probably need a comma at the end of the line?

**Comment by ansonla3** [comment]
still not merge ?

**Comment by lorensr** [comment]
No, sorry – there's a bug with the meteor/apollo beta release that we
haven't resolved yet

**Comment by lorensr** [comment]
`apollo@0.1.0` is out. Thanks again @jangerhofer!

@stubailo This is what GH is telling me to do:

image

Here is what the tree looks like after that:

image

Is there / should I use a prettier way? What are the commands?

**Comment by stubailo** [comment]
mm, try `git rebase master`?

**Comment by lorensr** [comment]
Thanks, how's this?

image

From my last point, I did:

```
git checkout master
git reset HEAD~
git checkout -- source/meteor.md
git checkout jangerhofer-patch-1
git rebase master
# edited out conflicts
git rebase --continue
git checkout master
git merge --no-ff jangerhofer-patch-1
```

**Comment by stubailo** [comment]
that seems reasonable to me. Any problems?

**Comment by lorensr** [comment]
Nope thanks, just wanted to check

**Comment by rdickert** [comment]
Thanks, @jangerhofer & @lorensr! 🎉

**Comment by lorensr** [comment]
Yikes, problem with this: https://github.com/apollostack/core-docs/pull/167#issuecomment-246060320

is that during the rebase, you have to re-fix conflicts after every single commit. Lot of repetition when there a multiple commits to the same conflicted area of the file.

**Pull request closed without merging**